controllers/workflow_controller.go

```diff
@@ -219,18 +221,11 @@ func (r *WorkflowReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 				workflow.Status.Reason = "DriverWait"
```

Comment:
The Status.Ready=False above  this can be removed.  It'll be set at the end of this function.